### DISCUSSION TITLE
APOE: Error from Monash with mapping diagnosis

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1508] [General]
![image](https://user-images.githubusercontent.com/43792259/193306331-3364ecb4-6ae3-4ef9-9822-9264b37f552f.png)

I'll confirm that there are no missing cells but the screenshot he sent looked good.

**Comment by rssk** [comment]
I might have an idea here, but I also think we fixed this? I'll triple check, but this could be an out of date app

**Reply by krootesmurdy** [reply]
I'll ask what version he is using, but I thought it updated automatically now?

**Reply by rssk** [reply]
it should but who knows

**Comment by dylanmartin** [comment]
Is it possible we could have the .csv and the compspec used when encountering this error?

**Reply by krootesmurdy** [reply]
![image](https://user-images.githubusercontent.com/43792259/193674182-1b4d7d93-f746-4eca-8e68-7715ee8cfae6.png)
Here's a screenshot they sent of the csv file

**Comment by pixelsaurus** [comment]
All that file path business needs to be removed. We just need to reference the actual file name and nothing else.

**Reply by krootesmurdy** [reply]
Sent!

**Reply by krootesmurdy** [reply]
Update: He needed the makespm12 script so I sent that along and then he will rename all the subjIDs once the files are moved.

**Reply by rssk** [reply]
the file paths _shouldn't_ break anything, but yeah in the future they just need to be relative to the csv. It would also probably be nice to move outputs to something less crazy path wise

**Comment by dylanmartin** [comment]
If this is related to mapping failing because of missing cells, it might be solved by this PR: 
https://github.com/trendscenter/coinstac/pull/1522
